draw-canvas.js

```diff
@@ -43,5 +50,10 @@ export default function main(pathname, backgroundImage) {
 
   board.draw();
 
-  return canv;
+  for (const { x, y, item } of input.tokens) {
+    renderer.render({ x, y, item });
+    // board.placeItem(x, y, item);
+  }
```

Comment:
any item can be rendered at an x,y position. The renderer will use the items type and name to determine which renderer to use.